mindjunkies/forums/models.py

```
@@ -34,76 +39,76 @@ def save(self, *args, **kwargs):
+    
+    def __str__(self):
+        return f'{self.user.username} : {self.comment.body[:30]}'
+    
+class Reply(models.Model):
+    author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL,null=True, related_name="replies")
+    parent_comment = models.ForeignKey(ForumComment, on_delete=models.CASCADE, related_name="replies",null=True,blank=True)
+    parent_reply=models.ForeignKey('Reply', on_delete=models.CASCADE, related_name="replies",null=True,blank=True)
+    body = models.CharField(max_length=150)
+    likes = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='likedreplies', through='LikedReply')
+    created = models.DateTimeField(auto_now_add=True)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)